src/group_theory/order_of_element.lean

```diff
@@ -323,11 +322,6 @@ begin
   exact ⟨k, hk⟩,
 end⟩
 
-omit hp
--- An example on how to determine the order of an element of a finite group.
-example : order_of (-1 : ℤˣ) = 2 :=
-order_of_eq_prime (int.units_sq _) dec_trivial
-
```

Comment:
Could you add this example to some other file instead?

Reply:
Done.